packages/signature-controller/src/SignatureController.ts

```diff
@@ -432,7 +431,7 @@ export class SignatureController extends BaseController<
         );
 
         resultCallbacks = acceptResult.resultCallbacks;
-      } catch {
+      } catch (e) {
```

Comment:
Very minor, but I think we try and avoid abbreviation so this could also be `error`.

Reply:
done